README.md

```diff
@@ -29,8 +29,8 @@ Kubernetes Fury Networking provides the following packages:
 
 | Package                    | Version  | Description                                                                                                                                          |
 | -------------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [calico](katalog/calico)   | `3.26.3` | [Calico][calico-page] CNI Plugin. For cluster with `< 50` nodes.                                                                                     |
-| [cilium](katalog/cilium)   | `1.14.3` | [Cilium][cilium-page] CNI Plugin. For cluster with `< 200` nodes.                                                                                    |
+| [calico](katalog/calico)   | `3.27.0` | [Calico][calico-page] CNI Plugin. For cluster with `< 50` nodes.                                                                                     |
+| [cilium](katalog/cilium)   | `1.15.0` | [Cilium][cilium-page] CNI Plugin. For cluster with `< 200` nodes.                                                                                    |
 | [tigera](katalog/tigera)   | `1.30.7` | [Tigera Operator][tigera-page], a Kubernetes Operator for Calico, provides pre-configured installations for on-prem and for EKS in policy-only mode. |
```

Comment:
The version of the Tigera operator is not the same as the Calico version it manages.
I suggest using the latest version of the operator.

```suggestion
| [tigera](katalog/tigera)   | `1.33.0` | [Tigera Operator][tigera-page], a Kubernetes Operator for Calico, provides pre-configured installations for on-prem and EKS in policy-only mode. |
```

Look at its repository: https://github.com/tigera/operator/releases/tag/v1.33.0

Reply:
You are right, but the README still has `v1.30.7`. You can change the version to `v1.32.3` if it's okay to use that version.
I was suggesting to upgrade to v1.33.0 since it's available from the project upstream

Reply:
Set as v1.32.3